framework/src/transfers/MultiAppNearestNodeTransfer.C

```diff
@@ -352,8 +352,8 @@ MultiAppNearestNodeTransfer::execute()
 
       if (_fixed_meshes)
       {
-        _cached_froms[pid].resize(qps.second.size());
-        _cached_dof_ids[pid].resize(qps.second.size());
+        _cached_froms[pid].resize(qps.second.size(), libMesh::invalid_uint);
+        _cached_dof_ids[pid].resize(qps.second.size(), DofObject::invalid_id);
```

Comment:
For thoroughness should we not instead do a fill after for all entries with invalid vals?

Reply:
I think in general there is no guarantee that the valid/invalid vector entries will be contiguous. There could be valid and invalid entries interlaced throughout the vector.

I'm not sure if that's what you were asking about/getting at though.

Reply:
What I mean is shouldn't we instead do:

```
auto & froms = _cached_froms[pid];
froms.resize(qps.second.size());
std::fill(froms.begin(), froms.end(), libMesh::invalid_uint);

auto & dof_ids = _cached_dof_ids[pid];
dof_ids.resize(qps.second.size());
std::fill(dof_ids.begin(), dof_ids.end(), DofObject::invalid_id);
```

Reply:
Yes that is a good idea ... that should be helpful if we ever do adaptivity in between these executes